### PR TITLE
Pic 3359 fix feature toggling

### PR DESCRIPTION
**Pull request by rschallapalli**

### CODE REVIEW

File: server/utils/features.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ const features = {
     ),
     allOf(
       isEnv('preprod'),
-      enabledForCourts(PLYMOUTH_MAGS_COURT_CODE, 'B05PK'),
```
Comment (paddybasi) [inline server/utils/features.js:20]: is it no longer a requirement to have it enabled for plymouth?

**Review by paddybasi — DISMISSED** [review]
LGTM

File: server/utils/features.js (hunk 1 above)
Comment (rschallapalli) [reply server/utils/features.js:20]: preprod - so removed court requirement to simplify granting access to the team

**Review by johnyfarrar — APPROVED** [review]
LGTM

**Pull request merged**